src/main/java/com/worktracker/model/Project.java

```diff
@@ -0,0 +1,15 @@
+package com.worktracker.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "projects")
+public class Project {
+
+    @Id
+    private int id;
```

Comment:
Use wrapper class and use `Long`

Reply:
I agree to use the wrapper class but I think the type should be Integer bacause this will always be 5-digit number